jenkins-design-language/gulpfile.js

```diff
@@ -246,9 +183,20 @@ var builder = require('@jenkins-cd/js-builder');
 builder.src([
     'src/js',
     'less',
-    'dist' // for icons & fonts
+    'dist' // for icons & fonts; NOTE: would be nice to find another way to do this as files in dist creates issues for jest & eslint
 ]);
 
+builder.tests('test/js');
```

Comment:
You have to define that because we do not use the "standard" location? If not since gulp-runner 1.1.0 you should not need this. We should fall back to the declared tasks in jsBuider

Reply:
Yes, this is used because the project structure of JDL and core-js are subtly different than dashboard, personalization, etc.